server/models/MatchModel.js

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
You can discuss this in your standup if it's outside of the scope of your task, but there could probably be more useful information added to this model